works.html

```diff
@@ -71,7 +71,300 @@
         </div>
       </header>
       <main class="main">
-        <!-- Start your work here ... -->
+        <div class="main__global-container">
+          <div class="main__global-container-title">
+            <h2 class="title-heading">Буровые работы</h2>
```

Comment:
Я не уверен, что h2 можно вообще по БЭМ-у делать блоком. Как правило, h2 иногда отличаются по размеру шрифта, т.е. они иногда разные. А ещё от этих h2 очень часто приходится делать margin-bottom вниз, чтобы отодвинуть остальной контент, а по БЭМ-у внешние отступы для блоков недопустимы. Плюс у тебя на станице этот блок только 1 раз и использовался, немного не понятно, почему ты сделала его блоком, а не элементом. Следовало написать class="main__title-heading". Я не стал исправлять, оставил так. Кстати, в scss, я бы вынес стилизацию блоков за пределы main, чтоб они отдельно были от элементов, но это не критично.
```suggestion
            <h2 class="main__title-heading">Буровые работы</h2>
```